istioctl/cmd/authz.go

```diff
@@ -201,80 +232,116 @@ func getConfigDumpFromPod(podName, podNamespace string) (*configdump.Wrapper, er
 	return envoyConfig, nil
 }
 
-func newConverter(v1PolicyFiles, serviceFiles []string, istioNamespace, istioMeshConfigMapName string) (*converter.Converter, error) {
-	v1alpha1Policies, err := getV1alpha1Policies(v1PolicyFiles)
+func createAuthorizationPoliciesFromFiles(files []string, rootNamespace string) (*model.AuthorizationPolicies, error) {
```

Comment:
Could we move this logic to a pkg owned by the security team because the UX team really isn't qualified to review this.

Reply:
Thanks for the review.
I also want to move more code to be owned by security team so that we can make the review process faster. Is it okay for me to do this in a follow up PR? This PR is already huge and @liminw had already reviewed and approved it.

Reply:
SGTM, I'll purely review from a UX perspective.